modules/transport-netty4/src/test/java/org/elasticsearch/transport/netty4/NetUtilsTests.java

```diff
@@ -12,13 +12,18 @@
 import org.elasticsearch.core.IOUtils;
 import org.elasticsearch.test.ESTestCase;
 
+import java.nio.channels.SocketChannel;
+
 public class NetUtilsTests extends ESTestCase {
 
-    public void testExtendedSocketOptions() {
+    public void testExtendedSocketOptions() throws Exception {
         assumeTrue("JDK possibly not supported", Constants.JVM_NAME.contains("HotSpot") || Constants.JVM_NAME.contains("OpenJDK"));
```

Comment:
Should we/can we remove this? As per your PR description, it should be supported on all JDKs?

Reply:
Yes, good point. Removed. It's a distro derived from OpenJDK or nothing!